File: egghead.io/data-structures-and-algorithms-in-js/lesson-12-sort-an-arr-with-a-nested-for-loop-using-insertion-sort-in-js/insertion-sort.js
const insertionSort = (array) => {
  for (let i = 1; i < array.length; i++) {
    for (let j = 0; j < i; j++) {
      console.log(array);

      if (array[j] > array[i]) {
        const [curr] = array.splice(i, 1);
        array.splice(j, 0, curr);
      }
    }
  }

  return array;
};

module.exports = insertionSort;

File: egghead.io/data-structures-and-algorithms-in-js/lesson-12-sort-an-arr-with-a-nested-for-loop-using-insertion-sort-in-js/insertion_sort.py
def insertion_sort(array):
  for i in range(1, len(array)):
    for j in range(0, i):
      print(array)

      if array[j] > array[i]:
        curr = array[i]
        array.remove(curr)
        array.insert(j, curr)
  
  return array